lib/bootstrap_form/form_builder.rb

```diff
@@ -58,14 +58,14 @@ def initialize(object_name, object, template, options)
                          options[:inline_errors] != false
                        end
       @acts_like_form_tag = options[:acts_like_form_tag]
-      add_form_role_and_form_inline options
+      add_default_form_attributes_and_form_inline options
       super
     end
     # rubocop:enable Metrics/AbcSize
 
-    def add_form_role_and_form_inline(options)
+    def add_default_form_attributes_and_form_inline(options)
       options[:html] ||= {}
-      options[:html][:role] ||= "form"
+      options[:html].reverse_merge!(BootstrapForm.config.default_form_attributes)
```

Comment:
It would be nice to make this work reasonably if the user sets `default_form_attributes = nil`. Extra bonus marks if we nicely handle if the user sets `default_form_attributes` to something other than a hash. Throwing an error on start-up might be better than waiting until the app actually tries to render a form.